openedx/features/custom_fields/multiselect_with_other/db/fields.py

```diff
@@ -104,6 +105,11 @@ def to_python(self, value):
         return MSFList(choices, [])
 
     def _check_other_max_length_attribute(self, **kwargs):
```

Comment:
Lets suppress linting warning and add following line two spaces after `**kwargs):` 

`# pylint: disable=unused-argument`